internal/commands/test.go

```diff
@@ -196,6 +196,27 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 				}
 			} else {
 				namespaces = viper.GetStringSlice("namespace")
+				if len(namespaces) == 1 {
+					regexp, err := regexp.Compile(namespaces[0])
```

Comment:
I couldn't see a way to check if the string contained regex, without parsing it, i.e.: "isRegex" method